cves/CVE-2013-0848.yml

```diff
@@ -259,10 +268,10 @@ interesting_commits:
       * A change that looks really sketchy
       * Commits where a developer mentions security in their message or comments
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+    - commit:
```

Comment:
I don't think this matters while parsing but if you're getting any errors watch out for these spaces.